CrowdFundingApplication/Views/Home/MainPage.cshtml

```diff
@@ -1,5 +1,104 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-*@
-@{
+﻿@{
+    ViewData["Title"] = "Главная";
 }
+
+<br />
+<br />
+
+<div class="mt-4">
+    <h3>Рекомендуемые проекты</h3>
+    <hr class="mb-4" />
+
+    <div class="campaign-card-list ml-auto mr-0">
+
+        <ul class="campaign-card-list_list ">
+
+            <li class="campaign-card-list_i">
+                <div class="card h-100" style="width: 18rem;">
+
+                    <div class="cont">
+                        <img src="~/images/img1.jpg" class="card-img-top image">
+                        <div class="overlayfade">
+                            <div class="text">Поддержать проект</div>
+                        </div>
+                    </div>
+
+                    <div class="card-body">
+                        <a class="small" href="#">Категория</a>
+                        <h6 class="card-title mt-1">Название</h6>
+                        <p class="card-text">Краткое описание проекта для которого необходимо собрадь деньги. Просто обычное описание какого-то проекта</p>
+                    </div>
+
+                    <div class="card-footer">
+                        <div class="progress" style="height: 3px;">
+                            <div class="progress-bar" role="progressbar" style="width: 25%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
+                        </div>
+                        <br />
```

Comment:
BR is added only in solid text to break lines

Reply:
Has been deleted